### PR TITLE
tests: add rule type check for flowbits v7

### PR DESCRIPTION
**Pull request by hadiqaalamdar**
Task #6309

Redmine ticket: https://redmine.openinfosecfoundation.org/issues/6309
Previous PR: https://github.com/OISF/suricata-verify/pull/1526

Suricata PR: https://github.com/OISF/suricata/pull/10018

Output from console:
```
===> flowbits: Sub test #11: FAIL : expected 1 matches; got 0 for filter {'filename': 'rules.json', 'count': 1, 'match': {'id': 11, 'lists.packet.matches[0].name': 'flowbits', 'lists.packet.matches[0].flowbits.action': 'noalert'}}
```
Output from stdout file:
```
Notice: suricata: This is Suricata version 7.0.3-dev (2fe2d8250 2023-10-19) running in USER mode [LogVersion:suricata.c:1149]
Warning: threshold-config: Error opening file: "/usr/local/etc/suricata//threshold.config": Permission denied [SCThresholdConfInitContext:util-threshold-config.c:177]
Warning: detect-flowbits: flowbit 'fb3' is checked but not set. Checked in 4 and 1 other sigs [DetectFlowbitsAnalyze:detect-flowbits.c:602]
Warning: detect-flowbits: flowbit 'fb4' is checked but not set. Checked in 5 and 0 other sigs [DetectFlowbitsAnalyze:detect-flowbits.c:602]
Warning: detect-flowbits: flowbit 'fb5' is checked but not set. Checked in 13 and 0 other sigs [DetectFlowbitsAnalyze:detect-flowbits.c:602]
```

### CODE REVIEW

**Review by jufajardini — CHANGES_REQUESTED** [review]
Or operand is there now \o\

Thanks for changing the `min-version` :)

I've noticed that the output example must be updated:
- it's indicating Suri 7, but this work is on Suri 8.
- we know that the other checks pass, so would be a good thing (and it is a good practice, too), to include an output sample of the eve rules file, showing how things are looking (although that might be a better case for the Suri PR, now that I think about it)
- What do you think of including at least one "ored" rule with three flowbits, just to showcase? ;)

**Comment by hadiqaalamdar** [comment]
New PR: https://github.com/OISF/suricata-verify/pull/1532

File: tests/rules/flowbits/test.yaml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+      lists.packet.matches[0].flowbits.names[1]: "fb6"
+- filter:
+    filename: rules.json
+    count: 1
+    match:
+      id: 7
+      lists.packet.matches[0].name: "flowbits"
+      lists.packet.matches[0].flowbits.cmd: "isnotset"
+      lists.packet.matches[0].flowbits.names[0]: "fb1"
+      lists.packet.matches[0].flowbits.names[1]: "fb2"
```
[the remark this answers is not in the document]
Comment (hadiqaalamdar) [reply tests/rules/flowbits/test.yaml:69]: sorry, I missed this comment. I'm not sure I understand how you want me to change the "ored" rules.

[the remark this answers is not in the document]
Comment (jufajardini) [reply tests/rules/flowbits/test.yaml:69]: Nothing to change in the rules themselves, just to add a check for the operand, for the cases when we expect to see it. :)

[the remark this answers is not in the document]
Comment (hadiqaalamdar) [reply tests/rules/flowbits/test.yaml:69]: do you mean adding lists.packet.matches[0].flowbits.operand: "or" in the test.yaml file?

[the remark this answers is not in the document]
Comment (jufajardini) [reply tests/rules/flowbits/test.yaml:69]: Yes!

**Pull request closed without merging**